pagopa/io-app
# chore: [IOBP-405] Wallet details update services pagoPA capability

## Short description
This PR implements a toggle to update the pagoPA capability status inside the wallet details screen;

## List of changes proposed in this pull request
- Added a new action `walletDetailsPagoPaCapabilityToggle`
- Adapted the `WalletDetailsPagoPaPaymentCapability` component;
- Added a new saga `handleTogglePagoPaCapability` that manages the toggle of the services that have `PAGOPA` as the service name, updating the status to `DISABLED` or `ENABLED`
- 
## How to test
- Checkout this PR from io-dev-api-server: https://github.com/pagopa/io-dev-api-server/pull/331
- Complete the new wallet onboarding process from the `New Wallet Playground`
- After you add a payment method correctly, you should be able to see the wallet details with a switch that can be toggled

## Preview

https://github.com/pagopa/io-app/assets/34343582/7884038e-97ab-444e-a02f-e8073e04f1e1
